modules/graphman-operation-validate.js

```diff
@@ -0,0 +1,40 @@
+const utils = require("./graphman-utils");
+const graphman = require("./graphman");
+var error = false;
+module.exports = {
+    run: function (params) {
+        if (params.input) {
+            const obj = utils.readFile(params.input)
```

Comment:
Please consider the input file as a valid graphman bundle.
Traverse through the entities and validate the policy code iterating through services and policies entities.

Reply:
done